html/views/overlay/index.js

```diff
@@ -81,18 +81,32 @@ function initialize() {
 		var style;
 		for(i in document.styleSheets) if(document.styleSheets[i].title =='jsStyle') style=document.styleSheets[i];
 		if(style) {
-			var ns,r;
+			var ns,r,ns2;
 			var rule;
 			// chrome seems to like things in lowercase
 			var rd = '#sb .colourteam'+k.Team;
 
-			if(k.Color == 'overlay_bg') ns = 'background-color';
+			if(k.Color == 'overlay_bg') { ns = 'background'; ns2 = 'background-color'; }
 			if(k.Color == 'overlay_fg') ns = 'color';
+
 			for(var r=0; r<style.rules.length ; r++ ) {
 				var dd = style.rules[r];
-				if(dd.selectorText == rd && dd.style[0] == ns) style.deleteRule(r);
+				if(dd.selectorText == rd && dd.style[0] == ns) {
+					style.deleteRule(r);
+				}
 			}
-			if(v != null) style.addRule(rd, ns + ': ' + v);
+
+			for(var r=0; r<style.rules.length ; r++ ) {
```

Comment:
This is getting messy.  Can we switch this to working like the PLT does?

Reply:
I replaced that now